RudderMoEngage.podspec

```diff
@@ -1,12 +1,12 @@
 Pod::Spec.new do |s|
     s.name             = 'RudderMoEngage'
-    s.version          = '1.0.0'
+    s.version          = '1.1.0'
```

Comment:
Please bump the version to `2.0.0` in all the files including Changelog, as you've updated the MoEngage API which might be a breaking change.

Reply:
done